packages/wonder-blocks-tooltip/components/tooltip.integration.test.js

```diff
@@ -0,0 +1,106 @@
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
+        // There's a 100ms delay before TooltipAnchor calls _setActiveState with
+        // instant set to true.  This second call is what actually triggers the
+        // call to this.props.onActiveChanged() which updates Tooltip's active
+        // state.
+        jest.runAllTimers();
+        // Since the call to update Tooltip's active state happens in a timeout
+        // we need to call update b/c it happens outside of the normal React lifecycle
+        // methods.
+        wrapper.update();
+
+        // Assert
+        expect(wrapper).toContainMatchingElement(TooltipBubble);
+        expect(wrapper).toHaveState("timeoutID", null);
+    });
+
+    it("should set a timeout on mouseleave on TooltipAnchor", () => {
+        // Arrange
+        const wrapper = mount(
+            <Tooltip content="hello, world">an anchor</Tooltip>,
+        );
+        const anchor = wrapper.find(TooltipAnchor).getDOMNode();
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
+        jest.runAllTimers();
+        wrapper.update();
+
+        // Act
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseleave"));
+        wrapper.update();
+
+        // Assert
+        expect(wrapper.state("timeoutID")).toEqual(expect.any(Number));
+        expect(wrapper.state("active")).toEqual(true);
+    });
+
+    it("should disable the timeout if the mouse hovers over TooltipAnchor within the TooltipDisappearanceDelay", () => {
+        // Arrange
+        const wrapper = mount(
+            <Tooltip content="hello, world">an anchor</Tooltip>,
+        );
+
+        const anchor = wrapper.find(TooltipAnchor).getDOMNode();
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
+        jest.runAllTimers();
+        wrapper.update();
+
+        const bubbleWrapper = wrapper.find(TooltipBubble);
+
+        // Act
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseleave"));
+        jest.advanceTimersByTime(TooltipDisappearanceDelay / 2);
+        wrapper.update();
+
+        bubbleWrapper.simulate("mouseenter");
+        wrapper.update();
+
+        // Assert
+        expect(wrapper.state("timeoutID")).toEqual(null);
+        expect(wrapper.state("active")).toEqual(true);
+    });
+
+    it("should close TooltipBubble on mouseleave on TooltipBubble", () => {
+        // Arrange
+        const wrapper = mount(
+            <Tooltip content="hello, world">an anchor</Tooltip>,
+        );
+
+        const anchor = wrapper.find(TooltipAnchor).getDOMNode();
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
+        jest.runAllTimers();
+        wrapper.update();
+
+        const bubbleWrapper = wrapper.find(TooltipBubble);
+
+        // Act
+        bubbleWrapper.simulate("mouseleave");
+
+        // Assert
+        expect(wrapper.state("timeoutID")).toEqual(null);
+        expect(wrapper.state("active")).toEqual(false);
```

Comment:
Could you add a `expect` here to verify that `TooltipBubble` is not longer being displayed?